src/components/Projects/WBS/wbs.jsx

```diff
@@ -30,11 +71,12 @@ const WBS = props => {
               </button>
             </NavItem>
 
-            <div id="member_project__name">Projects</div>
+            <div id="member_project__name">Return to Projects</div>
+            <div className="wbs-title">{projectName}</div>
           </ol>
         </nav>
 
-        <AddWBS projectId={projectId} />
+        <AddWBS projectId={projectId} toggleSort={toggleSort} />
```

Comment:
`toggleSort` doesn't align with the purpose of the `<AddWBS />` component. There should be a component dedicated to toggle sorting status. Leaving it this way would be unintuitive to further develop.